Sources/NIOCore/ByteBuffer-aux.swift

```diff
@@ -88,6 +88,30 @@ extension ByteBuffer {
         )
     }
 
+    // MARK: Hex encoded string APIs
+    /// Write `string` into this `ByteBuffer` in ASCII hexadecimal, moving the writer index forward appropriately.
+    /// - parameters:
+    ///     - plainHexEncodedBytes: The hex encoded string to write.
+    /// - returns: The number of bytes written.
```

Comment:
Can this doc comment be updated to have an example of what this format looks like?

Reply:
LMK if latest push meets expectation.